src/pages/categories/Categories.tsx

```diff
@@ -106,13 +107,20 @@ const Categories = () => {
         />
       </AppToolbar>
 
-      <CardsList
-        btnText={t('categoriesPage.viewMore')}
-        cards={cards}
-        isExpandable={isExpandable}
-        loading={categoriesLoading}
-        onClick={loadMore}
-      />
+      {categories.length > 0 ? (
```

Comment:
you can use categories.length without > 0

Reply:
Fixed